pkg/controller/migmigration/task.go

```diff
@@ -145,6 +145,13 @@ func (t *Task) Run() error {
 		t.Phase = RestoreCompleted
 	}
 
+	if t.Phase == RestoreCompleted && t.stage() {
```

Comment:
You don't need (or want) to check the phase here.  If the run() get this far, the restore has completed.

Reply:
Ah okay thanks Jeff